js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -0,0 +1,145 @@
+import {useStateWithStorage} from '@dagster-io/dagit-core/hooks/useStateWithStorage';
+import {
+  Body,
+  Box,
+  Button,
+  Checkbox,
+  Colors,
+  Dialog,
+  ExternalAnchorButton,
+  Heading,
+  Icon,
+  TextInput,
+} from '@dagster-io/ui';
+import React from 'react';
+import isEmail from 'validator/lib/isEmail';
+
+export const CommunityNux = () => {
+  const [didDismissCommunityNux, dismissCommunityNux] = useStateWithStorage(
+    'communityNux',
+    (data) => data,
+  );
+  if (didDismissCommunityNux) {
+    return null;
+  }
+  return (
+    <CommunityNuxImpl
+      dismiss={() => {
+        dismissCommunityNux('1');
+      }}
+    />
+  );
+};
+
+const CommunityNuxImpl: React.FC<{dismiss: () => void}> = ({dismiss}) => {
+  const [shouldShowNux, setShouldShowNux] = React.useState(false);
+  React.useEffect(() => {
+    const timeout = setTimeout(() => {
+      setShouldShowNux(true);
+    }, 5000); // wait 5 seconds before showing Nux
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, []);
+
+  const [email, setEmail] = React.useState('');
+  const [newsletter, setNewsLetter] = React.useState(false);
+
+  const submit = async () => {
+    await fetch('http://dagster.cloud/email-sign-up', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email,
+        newsletter,
+      }),
+    });
+    dismiss();
+  };
```

Comment:
Does this POST to `dagster.cloud` actually work? I kind of feel like this should be a GraphQL mutation, instead of posting to another domain.

Reply:
Yes, we want this data to go to the cloud endpoint so that we don't need to give postgress write access to another server.

Reply:
this should be `https` right?